### PR TITLE
Issues #118, #119. Add suite2p/facemap roadmaps

### PR DESCRIPTION
**Pull request by CBroz1**

### CODE REVIEW

**Comment by CBroz1** [comment]
@kabilar - Please review the links I added for the Facemap Development Roadmap. These were missing from the google doc.

File: docs/community/suite2p.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+## Development Roadmap
+
+- Mechanism to import Suite2p results - Completed 2021 - 
+[Element Interface Suite2p module](https://github.com/datajoint/element-interface/blob/main/element_interface/suite2p_loader.py)
+
+- Mechanism to run Suite2p within DataJoint Element - Completed 2022 - 
+[Element Calcium Imaging](https://github.com/datajoint/element-calcium-imaging/blob/00df4434fcfd6c1497d7950601248f046170139e/element_calcium_imaging/imaging.py#L267-L299)
+
+- Tutorials on running DataJoint Element with Suite2p - Completed 2022 -
```
Comment (kabilar) [inline docs/community/suite2p.md:89]:
```suggestion
- Tutorials on running DataJoint Element with Suite2p - Completed 2021 -
```

File: docs/community/facemap.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+### Licensing
+
+The two parties ensure that relevant software components are developed under licenses
+that avoid any hindrance to integration and interoperability between DataJoint Elements
+workflows and Facemap utilities.
+
+## Development Roadmap
+
+- Mechanism to import Facemap results - Completed 2022 - 
+[Link](https://github.com/datajoint/element-facemap/blob/main/element_facemap/facial_behavior_estimation.py#L389-L405)
```
Comment (kabilar) [inline docs/community/facemap.md:84]:
```suggestion
[Element Facemap](https://github.com/datajoint/element-facemap/blob/0ccab4ec6731cd612e7cf61a221c64fb9bf22566/element_facemap/facial_behavior_estimation.py#L389-L405)
```

File: docs/community/facemap.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+- Mechanism to run Facemap within DataJoint Elements - Completed 2022 - 
+[Link](https://github.com/datajoint/element-facemap/blob/main/element_facemap/facial_behavior_estimation.py#L259-L266)
```
Comment (kabilar) [inline docs/community/facemap.md:87]:
```suggestion
[Element Facemap](https://github.com/datajoint/element-facemap/blob/0ccab4ec6731cd612e7cf61a221c64fb9bf22566/element_facemap/facial_behavior_estimation.py#L259-L266)
```

File: docs/community/facemap.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+**DataJoint Elements** — https://elements.datajoint.org — is a collection of open-source
+  reference database schemas and analysis workflows for neurophysiology experiments, 
+  supported by **DataJoint** — https://datajoint.org — an open-source software 
+  framework. The project is funded by the NIH grant U24 NS116470 and led by Dr. Dimitri 
+  Yatsenko.
+  
+The principal developer of DataJoint Elements and the DataJoint framework is the company
+DataJoint — https://datajoint.com.
+
+### Neurodata without Borders (Facemap)
```
Comment (kabilar) [inline docs/community/facemap.md:30]:
```suggestion
### Facemap
```

File: docs/community/facemap.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+**Facemap** - https://github.com/MouseLand/facemap — is a pipeline for processing
+calcium imaging data. The project is funded by HHMI Janelia Research Campus and led by
```
Comment (kabilar) [inline docs/community/facemap.md:33]:
```suggestion
imaging data. The project is funded by HHMI Janelia Research Campus and led by
```

File: docs/community/facemap.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+Dr. Carsen Stringer and Atika Syeda.
+
+The principal developers of Facemap are at Janelia Research Campus.
```
Comment (kabilar) [inline docs/community/facemap.md:36]:
```suggestion
The principal developers of Facemap are at the Janelia Research Campus.
```

File: docs/community/suite2p.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+The principal developer of DataJoint Elements and the DataJoint framework is the company
+DataJoint — https://datajoint.com.
+
+### Suite2p
+
+**Suite2p** — https://www.suite2p.org — is a pipeline for processing calcium imaging
+  data. The project is funded by HHMI Janelia Research Campus and led by Dr. Carsen
+  Stringer and Dr. Marius Pachitariu.
+
+The principal developers of Suite2p are at Janelia Research Campus.
```
Comment (kabilar) [inline docs/community/suite2p.md:36]:
```suggestion
The principal developers of Suite2p are at the Janelia Research Campus.
```

File: docs/community/facemap.md [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+To achieve the aims of coordinated development, the principal developers conduct a joint
+annual review of this roadmap document to ensure that the two programs are
+well integrated and not redundant. The contents and resolutions of the review will be
+made publicly available.
+
+### Licensing
+
+The two parties ensure that relevant software components are developed under licenses
+that avoid any hindrance to integration and interoperability between DataJoint Elements
+workflows and Facemap utilities.
```
Comment (kabilar) [inline docs/community/facemap.md:79]:
```suggestion
that avoid any hindrance to integration and interoperability between DataJoint Elements 
and Facemap.
```

File: docs/community/facemap.md [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+
+- Tutorials on running DataJoint Element with Facemap - Under development 
+
+- Integration tests to verify loading Facemap data - Under development
+
+- Integration tests to verify running Facemap - Under development
+
+## Citation
+
+If you use this package please cite
```
Comment (kabilar) [inline docs/community/facemap.md:97]:
```suggestion
If you use Facemap please cite 
```

**Review by kabilar — CHANGES_REQUESTED** [review]
Thanks @CBroz1!

**Comment by CBroz1** [comment]
@kabilar Thanks for your review. Ready to merge when you're back from PTO

**Review by kabilar — APPROVED** [review]
Thanks @CBroz1!

**Pull request merged**